CHANGELOG.md

```diff
@@ -32,7 +32,7 @@
 - Allow CxxModules to implement functions which take two callbacks ([8826d8b](https://github.com/facebook/react-native/commit/8826d8b) by [@acoates-ms](https://github.com/acoates-ms))
 
 #### Breaking Changes
-- Public methods of components converted to ES6 classes are no longer bound to their component instance. For `ScrollView`, the affected methods are `setNativeProps`, `getScrollResponder`, `getScrollableNode`, `getInnerViewNode`, `scrollTo`, `scrollToEnd`, `scrollWithoutAnimationTo`, and `flashScrollIndicators`. Therefore, it is no longer safe to pass these method by reference as callbacks to functions. Auto-binding of methods was a behaviour of `createReactClass` that we decided to not preserve when switching over to ES6 classes. 
+- Public methods of components converted to ES6 classes are no longer bound to their component instance. For `ScrollView`, the affected methods are `setNativeProps`, `getScrollResponder`, `getScrollableNode`, `getInnerViewNode`, `scrollTo`, `scrollToEnd`, `scrollWithoutAnimationTo`, and `flashScrollIndicators`. For `CameraRollView`, the affected methods are: `rendererChanged`. For `SwipeableRow`, the affected methods are: `close`. Therefore, it is no longer safe to pass these method by reference as callbacks to functions. Auto-binding methods to component instances was a behaviour of `createReactClass` that we decided to not preserve when switching over to ES6 classes.
```

Comment:
They are using arrow functions to bind to component instance, see https://github.com/facebook/react-native/commit/221e2fe4095bc9ae15878725bdac4071d53e61f5#diff-f8f8b220fc0d1573e4f8b78c84415075R613 , why do you say they are no longer bound to their component instance? If that is the case, I believe the fb core team would already have their own projects broken.